src/snekmate/governance/mocks/TimelockControllerMock.vy

```diff
@@ -0,0 +1,126 @@
+# pragma version ~=0.4.0b5
+"""
+@title TimelockController Module Reference Implementation
+@custom:contract-name TimelockControllerMock
+@license GNU Affero General Public License v3.0 only
+@author pcaversaccio
+"""
+
+
+# @dev We import and implement the `IERC165` interface,
+# which is a built-in interface of the Vyper compiler.
+from ethereum.ercs import IERC165
+implements: IERC165
+
+
+# @dev We import and implement the `IAccessControl`
+# interface, which is written using standard Vyper
+# syntax.
+from ...auth.interfaces import IAccessControl
+implements: IAccessControl
+
+
+# @dev We import and implement the `IERC721Receiver`
+# interface, which is written using standard Vyper
+# syntax.
+from ...tokens.interfaces import IERC721Receiver
+implements: IERC721Receiver
+
+
+# @dev We import and implement the `IERC1155Receiver`
+# interface, which is written using standard Vyper
+# syntax.
+from ...tokens.interfaces import IERC1155Receiver
+implements: IERC1155Receiver
+
+
+# @dev We import and initialise the `AccessControl` module.
+from ...auth import AccessControl as ac
+initializes: ac
+
+
+# @dev We import and initialise the `Ownable` module.
+from .. import TimelockController as tc
+initializes: tc[access_control := ac]
+
+
+# @dev We export (i.e. the runtime bytecode exposes these
+# functions externally, allowing them to be called using
+# the ABI encoding specification) all `external` functions
+# from the `TimelockController` module.
+# @notice Please note that you must always also export (if
+# required by the contract logic) `public` declared `constant`,
+# `immutable`, and state variables, for which Vyper automatically
+# generates an `external` getter function for the variable.
+exports: (
```

Comment:
Hopefully soon we will have `module.__all__` which exports all `external` functions (incl. getters) by default (ok maybe I'm gonna implement it myself if I have bandwidth...). Until then we do it the explicit way 😄